ui/ui-frontend/projects/vitamui-library/src/app/modules/models/criteria/search-criteria.service.ts

```diff
@@ -36,18 +36,30 @@
  */
 import { SearchCriteriaAddAction, SearchCriteriaTypeEnum } from './search-criteria.interface';
 import { CriteriaDataType, CriteriaOperator } from './criteria.enums';
-import { searchCriteriaConfigs } from './search-criteria-configs';
+import {
+  END_DATE_FIELDS,
+  FINAL_ACTION,
+  FINAL_ACTION_TYPE,
+  INTERVAL_DATE_FIELDS,
+  RULE_ORIGIN,
+  searchCriteriaConfigs,
+} from './search-criteria-configs';
 import { Injectable } from '@angular/core';
 import { SearchWithTypeSelectorValue } from '../../../../lib/components/search-with-type-selector/search-with-type-selector.component';
 
-export type ArchiveUnitType =
-  | 'ARCHIVE_UNIT_FILING_UNIT'
-  | 'ARCHIVE_UNIT_HOLDING_UNIT'
-  | 'ARCHIVE_UNIT_WITH_OBJECTS'
-  | 'ARCHIVE_UNIT_WITHOUT_OBJECTS'
-  | 'FINAL_ACTION_TYPE'
-  | 'ALL_ARCHIVE_UNIT_TYPES'
-  | string;
+export const keys = [
```

Comment:
```suggestion
export const translatedKeys = [
```
Et déplacer dans une constante dans search-criteria-configs.ts ?

Reply:
Done